frontend/src/app/render_wasm/api.cljs

```diff
@@ -533,7 +560,7 @@
                 (set-shape-svg-raw-content (get-static-markup shape)))
               (when (some? bool-content) (set-shape-bool-content bool-content))
               (when (some? corners) (set-shape-corners corners))
-              (let [pending' (concat (set-shape-fills fills) (set-shape-strokes strokes))]
+              (let [pending' (concat (set-shape-fills fills) (set-shape-strokes strokes) (set-shape-shadows shadows))]
```

Comment:
Is this needed?, shadows can't use image as fills so they don't generate pending network requests